common.blocks/image/image.en.md

```diff
@@ -105,7 +105,17 @@ Specifies the image height in pixels.
 
 Type: `BEMJSON`.
 
-Specifies content of the image in SVG format.
+Specifies content of the image in SVG format. Depends on `escapeContent` option
```

Comment:
Specifies content of the image in SVG format.

Usage depends on `escapeContent` option of `bem-xjst`.

For `escapeContent: true` (default behavior) it should be
```js
// ...
```

For `escapeContent: false` it should be
```js
// ...
```

Reply:
@tadatuta updated.